docs/relational-databases/performance/tune-performance-with-the-query-store.md

```diff
@@ -105,7 +112,25 @@ ORDER BY rs.avg_physical_io_reads DESC;
 
 #### Queries with multiple plans
 
-These queries are especially interesting because they're candidates for regressions due to plan choice change. The following query identifies these queries along with all plans:
+These queries are especially interesting because they're candidates for regressions due to plan choice change. 
+
+The following query identifies the queries with the highest number of plans:
```

Comment:
I would change this query - there's no predicate on it so it will run against the entire Query Store.  Also, it's not restricted so it won't return queries with the highest number of plans, it will return ALL queries that have more than one plan.  My suggestion below.

Also, please change "cnt" to "PlanCount" or something similar :)  It's also used in another query.  Thanks!

SELECT 
	q.query_id, 
	object_name(object_id) AS ContainingObject,
	COUNT(*) AS QueryPlanCount, 
	STRING_AGG( p.plan_id,',') plan_ids,
	qt.query_sql_text
FROM sys.query_store_query_text AS qt
JOIN sys.query_store_query AS q
    ON qt.query_text_id = q.query_text_id
JOIN sys.query_store_plan AS p
    ON p.query_id = q.query_id
JOIN sys.query_store_runtime_stats AS rs ON
   p.plan_id = rs.plan_id
WHERE rs.last_execution_time > DATEADD(HOUR, -1, GETUTCDATE())
GROUP BY OBJECT_NAME(object_id), q.query_id, qt.query_sql_text
HAVING COUNT(distinct p.plan_id) > 1
ORDER BY QueryPlanCount desc